pkgs/bufbuild/protoc-gen-validate/protoc-gen-validate-go/registry.yaml

```diff
@@ -0,0 +1,11 @@
+packages:
+  - type: go_install
```

Comment:
You should use `github_release`.
You don't have to build binaries yourself.
https://github.com/bufbuild/protoc-gen-validate/releases

https://aquaproj.github.io/docs/reference/registry-style-guide#select-type-according-to-the-following-order